src/locales/en/translation.json

```diff
@@ -99,6 +102,10 @@
     "title": "Oasis Protocol Foundation",
     "version": "Version: {{buildSha}}"
   },
+  "layerPickerModal": {
+    "title": "Paratime Details",
+    "description": "Below is the overview of available ParaTimes. Your search was performed across the active ParaTime"
```

Comment:
Searches are performed across all Paratimes. We also dont show this element on search pages. We should discuss if that's changing